static/app/views/organizationStats/teamInsights/teamReleases.tsx

```diff
@@ -208,22 +210,16 @@ function TeamReleases({
             },
           ]}
           tooltip={{
-            formatter: (seriesParams: any) => {
+            formatter: seriesParams => {
               // `seriesParams` can be an array or an object :/
               const [series] = toArray(seriesParams);
 
-              if (!series.data?.value) {
-                return '';
-              }
-
               const dateFormat = 'MMM D';
-              const startDate = moment(series.data.value[0]).format(dateFormat);
-              const endDate = moment(series.data.value[0])
-                .add(7, 'days')
-                .format(dateFormat);
+              const startDate = moment(series.data[0]).format(dateFormat);
+              const endDate = moment(series.data[0]).add(7, 'days').format(dateFormat);
               return [
                 '<div class="tooltip-series">',
-                `<div><span class="tooltip-label">${series.marker} <strong>${series.seriesName}</strong></span> ${series.data.value[1]}</div>`,
+                `<div><span class="tooltip-label">${series.marker} <strong>${series.seriesName}</strong></span> ${series.data[1]}</div>`,
```

Comment:
why are we undo-ing the changes here?

Reply:
i screwed up rebasing so there's some changes that aren't supposed to be here

Reply:
ok resolved, now back to figuring out how to squash the commits